src/client/sandbox/shadow-ui.js

```diff
@@ -254,29 +257,33 @@ export default class ShadowUI extends SandboxBase {
                 }, elContextWindow);
             }
             else
-                this.onBodyContentChanged();
+                this.makeRootLastBodyChild();
         });
 
         this.messageSandbox.on(this.messageSandbox.SERVICE_MSG_RECEIVED_EVENT, e => {
             if (e.message.cmd === this.BODY_CONTENT_CHANGED_COMMAND)
-                this.onBodyContentChanged();
+                this.makeRootLastBodyChild();
         });
     }
 
-    onBodyContentChanged () {
-        if (this.root) {
-            if (!domUtils.closest(this.root, 'html'))
-                this.nativeMethods.appendChild.call(this.document.body, this.root);
+    makeRootLastBodyChild () {
```

Comment:
I suppose it should be `onBodyElementMutation` (discussed in private with @helen-dikareva )